mock/py/mockbuild/util.py

```diff
@@ -1435,7 +1435,6 @@ def setup_operations_timeout(config_opts):
 def do_update_config(log, config_opts, cfg, uidManager, name, skipError=True):
     if os.path.exists(cfg):
         log.info("Reading configuration from %s", cfg)
-        config_opts['config_paths'].append(cfg)
```

Comment:
This removal requires #423 though.  Can you please merge it into one commit?

Reply:
Or perhaps at least move this chunk to #423

Reply:
No, it is distinct of #423, #423 is the initialization of config_paths , we can remove  config_paths.append(cfg) here because it will do the same  .append in include function  (line 1378) , for some reason the .append in include function doesn't depend on is_statement, as I though, but forget.

with this commit `mock -r fedora-30-x86_64 --debug-config` will be [1] while with current master [2]


[1]
config_opts['config_paths'] = ['/etc/mock/site-defaults.cfg',
 '/etc/mock/fedora-30-x86_64.cfg',
 '/home/sergio/.config/mock.cfg']

[2]
config_opts['config_paths'] = ['/etc/mock/site-defaults.cfg',
 '/etc/mock/site-defaults.cfg',
 '/etc/mock/fedora-30-x86_64.cfg',
 '/etc/mock/fedora-30-x86_64.cfg']

Reply:
Yes, exactly ... it was a real puzzle why you are doing that change, and I really think that
it logically belongs to #423 because of this (if it was in #423, nobody would ask those
questions I believe).

Reply:
So... maybe it deserves separate commit.

Reply:
I have merged #423